### ISSUE TITLE
[css-overflow-5] Default UA styles for scroll markers and scroll buttons

### ISSUE THREAD

**Issue by flackr** [w3c/csswg-drafts#10935]
When developers create scroll markers and scroll buttons, they should provide good default UA styles to be accessible and usable. With nothing spec'd it takes a lot of styles to get these controls usable.

I propose, scroll buttons be styled like buttons by the UA stylesheet. I'd suggest copying the UA button styles, including pseudo-class states (e.g. `:focus`).

Scroll markers behave as links, and as such I think should use the UA link styles. `:visited` will never apply to pseudo-element scroll markers, so that part can be skipped.

I think `::scroll-marker-group` should have a specified size large enough to fit a line of `::scroll-marker` elements, perhaps scrolling by default if the list is too large.

The end goal is that with no specific styling applied these controls should all work reasonably without any particular styling attention.

**Comment by flackr** [comment]
> [@tabatkins](https://github.com/tabatkins) do you know if / where these styles are specified to link to?

I did a search, and as far as I can tell, there is [no specification of what an `appearance: none` button looks like](https://www.w3.org/TR/css-ui-4/#control-specific-rules) in css-ui-4 and only a non-normative specification of [how links could be styled](https://www.w3.org/TR/css-color-4/#sample) in css-color-4, so this suggests that if we want these to align it should just make reference to being styled the same as buttons and links.

**Comment by tabatkins** [comment]
Right, there's no definition of the "primitive appearance" right now. It exists only in the browsers' own UA stylesheets.

I agree we should just say that they're styled "like buttons", with reference to button's "native appearance" and "primitive appearance".

**Comment by flackr** [comment]
Here's an [example page](https://jsbin.com/yofihuf/edit?html,css,output) that only sets the `content` property and otherwise uses the default UA styles:

<img width="414" alt="Image" src="https://github.com/user-attachments/assets/970ccc04-eebe-4528-950c-97b152e7609b" />

The `::scroll-button` pseudo-elements appear the same as buttons and the `::scroll-marker` pseudo-elements appear the same as links that never match `:visited`.
